main_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestIndexHandler(t *testing.T) {
+	req, err := http.NewRequest("GET", "/", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	rec := httptest.NewRecorder()
+	indexHandler(rec, req)
```

Comment:
Is that only in "scope" because we're in the same package as main.go? Or is it being imported in some other way?